File: cap05/src/main/java/app/car/cap05/interfaces/incoming/output/TravelRequestOutput.java
package app.car.cap05.interfaces.incoming.output;

import app.car.cap05.domain.TravelRequestStatus;

import java.time.LocalDateTime;
import java.util.Date;
import lombok.Data;

@Data
public class TravelRequestOutput {

    Long id;
    String origin;
    String destination;
    TravelRequestStatus status;
    LocalDateTime creationDate;


}

File: cap09/src/main/kotlin/app/car/car09/interfaces/incoming/output/TravelRequestOutput.kt
package app.car.car09.interfaces.incoming.output

import app.car.car09.domain.TravelRequestStatus
import java.util.Date

open class TravelRequestOutput(
        val id: Long,
        var origin: String,
        var destination: String,
        var status: TravelRequestStatus,
        var creationDate: Date
)